newsfragments/8667.feature.rst

```diff
@@ -0,0 +1 @@
+Added the export and import of recovery devices to help with password loss
```

Comment:
It's not only password loss, but also if the computer is stolen or broken, etc. 

```suggestion
Added support for recovery files to recover access in case the password is lost or the user loses access to its computer
```